ceph-releases/jewel/ubuntu/14.04/daemon/config.k8s.sh

```diff
@@ -1,9 +1,20 @@
 #!/bin/bash
 set -e
 
+# log arguments with timestamp
+function log {
```

Comment:
since you are adding a log function, can you make it so that we have different levels? no definition in levels is going to make it hard to understand what is informational vs error reporting